packages/buffetjs-styles/src/components/Button/index.js

```diff
@@ -62,18 +63,24 @@ const Button = styled.button`
       }
     `}
 
-  ${({ color }) =>
-    color === 'delete' &&
-    `
-      padding: 0 ${sizes.button.padding.small};
-      
-    `}
-
   ${({ color }) =>
     color === 'success' &&
     `
-      min-width: ${sizes.button.minWidth.small};
+      min-width: ${sizes.button.minWidth};
     `}
+  
+    // FontAwesome icons
+    > span svg {
```

Comment:
The svg selector is overridden by the second one so it not needed.

Reply:
It is needed because this one is about FontAwesome svgs, which are in a span, and the second one is for our custom svgs. The behavior is not the same for each and the second one isn't overridden because the selector is different.

Reply:
So which `vertical-align` property is needed for the `svg` ?

Reply:
As I said, for a direct children selected by `> span` and which is a Custom icons, it baseline. For a children of a span which is a FontAwesome icon (it's the way they are), it's initial.